packages/unhead/src/scripts/proxy.ts

```
@@ -2,66 +2,39 @@ import type { AsVoidFunctions, RecordingEntry } from './types'
     },
-    apply(_, __, args) {
-      stack[stackIdx].push({ type: 'apply', key: '', args })
-      return undefined
-    },
-  } as ProxyHandler<T>)
-
-  return {
-    proxy: new Proxy(instance || {}, handler()),
-    stack,
-  }
+    apply(_, __, args) { stack[stackIdx].push({ type: 'apply', key: '', args }); return undefined },
```

ESLint

[error] This line has 2 statements. Maximum allowed is 1.

(style/max-statements-per-line)